specs/agents/tracing-api-ot.md

```diff
@@ -0,0 +1,36 @@
+## OpenTracing API
```

Comment:
TBH, I wouldn't bother specifying the OpenTracing bridge.

Reply:
+1 to remove it too, that could simplify structure too.

Reply:
Alright, let's remove it from this PR then 🙂